Makefile

```diff
@@ -71,8 +71,15 @@ bundle-all-platforms:
 
 # Packages
 
-package-test:
-	cd $(ROOT_DIR) && go run ./main.go package test ./preflight-packages/examples.jetstack.io
+packages-lint:
+	cd $(ROOT_DIR)/.cache/cli/repo && \
+	go run . package lint $(ROOT_DIR)/preflight-packages/jetstack.io/pods && \
```

Comment:
We should make `package lint` match the behaviour of `package test` and make it search for packages recursively.

Reply:
(not necessarily as part of this PR)